labs/ipfs-naming/api-cli/ipfs-manage-keys.mdx

```diff
@@ -0,0 +1,58 @@
+---
+meta:
+title: How to manage your key
+description: This page shows you how to manage your key using kubo client
+content:
+h1: How to manage your key
+paragraph: This page shows you how to manage your key using kubo client
+tags: ipfs naming key name desktop labs web3 kubo
+dates:
+validation: 2023-05-25
+posted: 2023-05-30
+categories:
+  - labs
+  - naming
+  - ipfs-naming
+---
+
+To manage your keys, we use [Kubo](https://docs.ipfs.tech/install/command-line/), the official IPFS implementation in Golang.
+
+## Export key using a Kubo client
+
+To export your kubo client key and convert it to base64 for import into our naming service, you can use this sequence of commands.
+
+1. Run the following CLI command to list the keys you own:
+
+ ```
+    ipfs key list
+```
+
+2. Run the following CLI command to export the key to a privkey.pem file:
+
+ ```
+  ipfs key export myKey -o privkey.pem
+ ```
+
+<Message type="important">
+    The key must be exported in libp2p-protobuf-cleartext format, which is the default.
+</Message>
+
+3. Convert file contents to base64 for import into the naming service via the scw CLI.
+    ```
+     cat privkey.pem | base64
+    ```
+
+## Import key to Kubo client
+
+Once you've exported your key from our service, you'll get your key in base64. Here's how to import it into your kubo client.
```

Comment:
```suggestion
After exporting your key from our service, it will be provided to you in `base64` format. Follow these steps to import the key into your Kubo client.
```